### DISCUSSION TITLE
QField Toolbar has no Cloud option

### DISCUSSION THREAD

**Discussion by pross2022** [opengisch/QField#2881] [Q&A]
I'm trying to connect QGIS to QField Cloud, but there is no option to connect to the cloud in QGIS. This is the toolbar I have:
![image](https://user-images.githubusercontent.com/105311578/167698086-fa90d5c3-bfe3-4aa2-80ba-5393a24f5ab5.png)
 
This is the toolbar I am looking for:
![image](https://user-images.githubusercontent.com/105311578/167698455-7dd71a8e-1435-4575-aead-d353fd88cfc4.png)

I can the QField Sync plugin downloaded but it does not give the option to access the cloud.

**Comment by signedav** [comment]
Seems your QFieldSync Plugin is outdated. Go in QGIS to "Plugins" > "Manage and Install Plugins", find "QFieldSync" and update your plugin.

**Reply by suricactus** [reply]
Only to add, you need at least QGIS 3.16 to be able to install QFieldSync 4.x.

**Comment by pross2022** [comment]
Thank you, it's been upgraded and I have access to the cloud. I was using the the Madeira 3.4 LTR, now I am using 3.24 Tisler
